index.html

```diff
@@ -0,0 +1,233 @@
+      </p>
+    </div>
+
+    <form id="survey-form">
+      <div>
+        <label for="name" id="name-label" class="questions"
+          >Name
+          <input
+            class="questions_2"
+            id="name"
+            type="text"
+            placeholder="Enter you name"
+            required
+          />
+        </label>
+      </div>
+      <div>
+        <label for="email" id="email-label" class="questions"
+          >Email
+          <input
+            class="questions_2"
+            id="email"
+            type="email"
+            placeholder="Enter you email"
+            required
+          />
+        </label>
+      </div>
+
+      <div>
+        <label for="age" id="number-label" class="questions"
+          >Age
+          <input
+            id="number"
+            class="questions_2"
+            type="number"
+            placeholder="Age"
+            min="10"
+            max="99"
+            required
+          />
+        </label>
+      </div>
+
+      <label id="first-dropdown" class="questions"
+        >Which option best describes your current role?</label
+      >
+      <select id="dropdown" class="questions_2">
+        <option disabled selected value="">Select current role</option>
+        <option value="">Student</option>
+        <option value="">Full time job</option>
+        <option value="">Full time learner</option>
+      </select>
+
+      <label class="questions"
+        >Would you recommend freeCodeCamp to a friend?</label
+      >
+      <section id="radio-buttons">
+        <div>
+          <label class="label-radio" for="option_1">
+            Definitely
+            <input type="radio" id="taste_1" name="option" value="Definitely" />
+          </label>
+        </div>
+        <div>
+          <label class="label-radio" for="option_2"
+            >Maybe
+            <input type="radio" id="taste_2" name="option" value="Maybe" />
+          </label>
+        </div>
+        <div>
+          <label class="label-radio" for="option_3"
+            >Not Sure
+            <input type="radio" id="taste_2" name="option" value="Not Sure" />
+          </label>
+        </div>
+      </section>
+      <label>What is your favorite feature of freeCodeCamp?</label>
+
+      <select class="questions_2">
+        <option disabled selected value="">Select an option</option>
+        <option value="">Challenges</option>
+        <option value="">Projects</option>
+        <option value="">Community</option>
+        <option value="">Open Source</option>
+      </select>
+
+      <label class="questions"
+        >What would you like to see improved? (Check all that apply)</label
+      >
+      <div id="labels-checkboxes">
+        <label>
+          <input
+            class="input-checkbox"
+            type="checkbox"
+            value="11"
+            name="like"
+            id="Front-end Projects"
+          />Front-end Projects</label
+        >
+        <label>
```

Comment:
I suggest to not nest the input inside the label, It would be better to use them as blocks and then style those with CSS using flexbox so they appear in the same line 💪. This will improve a lot your working flow 😀